src/parser/warrior/protection/CHANGELOG.js

```diff
@@ -4,14 +4,19 @@ import React from 'react';
 import SPELLS from 'common/SPELLS';
 import SpellLink from 'common/SpellLink';
 
-import { Zerotorescue, Abelito75 } from 'CONTRIBUTORS';
+import { Abelito75 } from 'CONTRIBUTORS';
 import { change, date } from 'common/changelog';
 
-export default [
+export default [  
```

Comment:
I don't think you need to mention which class or spec the changes were made for, since these are only visible in the context of protection warrior changes. 

There's some inconsistency with your capitalization of words.

This is more of a style thing, but try keeping the presented text somewhat "professional". Stuff like "No more stone age!" doesn't belong here in my opinion. Also try to keep abbreviations fully capitalized (DPS instead of dps, POV instead of pov)